packages/fluence-js/src/__test__/_aqua/srv-tests.ts

```diff
@@ -0,0 +1,341 @@
+/**
```

Comment:
shouldn't it be possible to not edit import statements manually now that we have monorepo
And also I am curious to know why are imports pointing to v3 when there is a v4?

Reply:
Oh, that was due to manual editing. Btw, separating aqua into another repository sound reasonable. I'll consider doing that

Reply:
That won't work right now :(
aqua does import the following:

```
import {
    callFunction$$,
    registerService$$,
} from '@fluencelabs/fluence/dist/internal/compilerSupport/v4';
```

So it's not possible to extract aqua code into separate library and then import in in `@fluencelabs/fluence`. It will result into circular dependencies:

```
services <- fluence <- services
```